app/(service)/my-store/_components/emptyStore.tsx

```diff
@@ -10,9 +12,11 @@ export default function EmptyNotice() {
             내 가게를 소개하고 공고도 등록해 보세요.
           </div>
           <div className=" md:w-[346px] rounded-md justify-center items-center gap-2 inline-flex">
-            <Button className="px-5 py-2 font-bold md:h-12" background="bg-primary">
-              가게 등록하기
-            </Button>
+            <Link href="/my-store/edit">
```

Comment:
P3: `route.ts` 파일에 경로 추가해서, 상수로 사용해주세요

```suggestion
            <Link href={ROUTE_PATHS.STORE_EDIT}>
```